cylc/flow/task_job_mgr.py

```diff
@@ -815,7 +815,7 @@ def _poll_task_job_callback(self, workflow, itask, cmd_ctx, line):
                 itask, INFO, FAIL_MESSAGE_PREFIX + jp_ctx.run_signal,
                 jp_ctx.time_run_exit,
                 flag)
-        elif jp_ctx.run_status == 1:
```

Comment:
This conversion is not quite identical as it causes the `jp_ctx.time_run` evaluation to happen earlier in the `if/elif` block.

In this case we can revert to the way it was before and add an inline ignore comment.